iothub/device/src/InternalClient.cs

```diff
@@ -181,7 +181,7 @@ public void SetConnectionStatusChangeHandler(Action<ConnectionStatusInfo> status
         {
             if (Logging.IsEnabled)
                 Logging.Enter(this, callback, userContext, nameof(SetDesiredPropertyUpdateCallbackAsync));
-            
+
```

Comment:
No documentation changes for either DeviceClient or ModuleClient about the behavior of SendEventAsync requiring users to call OpenAsync first?

Reply:
Also missing documentation on what exception to expect if the user tries to perform a send operation when the client is closed

Reply:
Currently, re-opening after close does not work but it should after this: https://msazure.visualstudio.com/One/_workitems/edit/14095155

Added documentation that client can be re-opened.